src/compiler/config/test/validate-config.spec.ts

```diff
@@ -312,6 +312,16 @@ describe('validation', () => {
     expect(validated.diagnostics).toHaveLength(1);
   });
 
+  it('should warn when dist-custom-elements-bundle is found', () => {
+    userConfig.outputTargets = [
+      {
+        type: 'dist-custom-elements-bundle',
+      } as any,
+    ];
+    const validated = validateConfig(userConfig);
+    expect(validated.diagnostics).toHaveLength(1);
```

Comment:
I like this, and since we verified that `validated.diagnostics` has a value at index 0, what're your thoughts on verifying the message as well?
```suggestion
    expect(validated.diagnostics).toHaveLength(1);
    expect(validated.diagnostics[0].messageText).toBe(
      'dist-custom-elements-bundle is deprecated and will be removed in a future version. Use "dist-custom-elements" instead. If "dist-custom-elements" does not meet your needs, please add a comment to https://github.com/ionic-team/stencil/issues/3136.'
    );
```

Reply:
Yes, good point. Updating with the changed warning message from latest commit:

```suggestion
    expect(validated.diagnostics).toHaveLength(1);
    expect(validated.diagnostics[0].messageText).toBe(
      'dist-custom-elements-bundle is deprecated and will be removed in a future major version release. Use "dist-custom-elements" instead. If "dist-custom-elements" does not meet your needs, please add a comment to https://github.com/ionic-team/stencil/issues/3136.'
    );
```